### PR TITLE
Max header

### PR DESCRIPTION
**Pull request by MaxymKoz**
My workable site version
I corrected sizes on page in media tags

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,151 +1,310 @@
...
+        <div class="nav_menu">
+          <img class="menu_picture" src="./images/Menu.png" />
+          <div class="startup">
+            <li class="text-menu">Foundation</li>
+            <li class="text-menu2">Startup landing template</li>
+          </div>
+          <nav class="one">
+            <ul class="topmenu">
+              <li>
+                <a href="" class="active"
```
Comment (Nazar0n) [inline index.html]: `href`

File: index.html [hunk 2]
```diff
@@ -1,151 +1,310 @@
...
+                  >Library<span class="fa fa-angle-down"></span
+                ></a>
+                <ul class="submenu">
```
Comment (Nazar0n) [inline index.html:43]: `ul` using with `li`. Can you remove something?

File: index.html (hunk 1 above)
Comment (MaxymKoz) [reply index.html]: i don't have link for it
but in work i can paste link

File: index.html [hunk 3]
```diff
@@ -1,151 +1,310 @@
...
+                  <div class="menu-choose">
+                    <div class="acquire">
+                      <span
+                        class="aquire-users"
+                        href="https://www.youtube.com/watch?v=s6my0RUoX4c"
+                        >Acquire Users
+                      </span>
+                      <br />
```
Comment (Nazar0n) [inline index.html]: Can you use flex instead of `br`

File: index.html [hunk 4]
```diff
@@ -1,151 +1,310 @@
...
+          <nav class="one">
+            <ul class="topmenu">
+              <li>
+                <a href="" class="active"
+                  >Library<span class="fa fa-angle-down"></span
+                ></a>
+                <ul class="submenu">
+                  <div class="menu-choose">
+                    <div class="acquire">
+                      <span
```
Comment (Nazar0n) [inline index.html]: Do you need `span` here?

File: styles/header.css [hunk 5]
```diff
@@ -0,0 +1,545 @@
...
+                                display: flex;
+                                box-sizing: border-box;
+                            }
+                            body{flex-direction: column;}
+                            
+                            .max_code {
+                                
+                                justify-content: center;
+                                width: 1400px;
+                                height: 29%;
```
Comment (Nazar0n) [inline styles/header.css]: I dont know, but maybe use `px` instead of %

**Pull request merged**